<launch>
  <arg name="node_name" default="picam1" />
  <!-- Number of stills to grab -->
  <arg name="num" default="1" />

  <node pkg="rosservice" type="rosservice" name="global_loc" args="call --wait /$(arg node_name)/stop_recording" />
</launch>
